providers.tf

```diff
@@ -1,16 +1,28 @@
 # This is the default provider that is used to create resources inside
-# the Users account
+# the Users account.
 provider "aws" {
 }
 
+# This is the provider that is used to create the role that can be
+# assumed to perform CI functions.
+provider "aws" {
+  alias = "images-production-provisionaccount"
+}
+
+# This is the provider that is used to create the role that can be
+# assumed to perform CI functions.
```

Comment:
...create the role **in the Images Staging account** that can be assumed...

Reply:
See [comment above](https://github.com/cisagov/molecule-iam-user-tf-module/pull/5#discussion_r426262036).